docs/dev/framework/front-end-modules.md

```diff
@@ -7,12 +7,6 @@ aliases:
     - /framework/front-end-modules/
 ---
 
-{{% notice note %}}
-This covers the documentation on how to create front end modules in Contao **4.6**
-and up. In previous Contao version, front end modules must extend from `\Contao\Module`
-and then be registered via the `$GLOBALS['FE_MOD']` array.
-{{% /notice %}}
-
```

Comment:
I don't think this should be removed as the old way is still supported in all Contao versions so far.

Reply:
Supported does not mean it needs to be documented.

Reply:
I think things that are common place and widely spread should be documented at least a little. It's not like we have a full documentation on legacy front end modules in any case. And since we will still support legacy modules in 5.0, I think this simple notice should stay.